plugins/dfa-interpreter.ts

```diff
@@ -0,0 +1,86 @@
+declare var Interpreter: any, InterpreterGraph: any, Program: any, ProgramInput: any, ProgramOutput: any, RunningProgram: any;
+//import { Graph, Node, Edge } from "../app/models/plugin";
```

Comment:
This is the correct (hack) approach right now, right? Since we don't want the plugins to reference the main project at all. We need someone to declare these models in a way that both sinap and plugins can compile with them.